### DISCUSSION TITLE
Grafana reserved labels in notification policies

### DISCUSSION THREAD

**Discussion by DaspawnW** [grafana/grafana#70787] [Alerting]
Is it possible to define within notification policies the `grafana_state_reason` as a label matcher?

The reason we are very unsure about this is that `rule_uid` is also not mentioned as label but it seems to work and is used by Grafana itself.
Based on the source code it looks like there is such a label https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/models/alert_rule.go#L106 that could be utilised on the other side checking the documentation of [reserved labels](https://grafana.com/docs/grafana/latest/alerting/fundamentals/annotation-label/how-to-use-labels/#grafana-reserved-labels) it only mentions `grafana_folder`.

An advice would be highly appreciated.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82589. Feel free to continue the discussion around this there. Thank you!
